docs/static_landing_page/index.html

```diff
@@ -1,307 +1,447 @@
-			<div class="mx-0">
-				<div class="row justify-content-lg-center pt-3 pb-5">
-				<div class="col-12 col-lg-8 text-center">
-					<h2 class="white pb-3">Contribute to Fairlearn</h2>
-					<p class="white pb-3">
-						Fairlearn is built and maintained by open source contributors with a variety of backgrounds and expertise.
-						Join the effort and contribute feedback, metrics, algorithms, visualizations, ideas and more, so we can evolve the toolkit together!
-					</p>
-					  <!-- Versioned Link -->
-					  <a class="contribute-btn mb-0 text-center" href="./v0.6.2/contributor_guide/index.html">Contributor Guide</a>
-				</div>
-			</div>
-		</div>
-		</div>
-	</div>
-	<!-- CONTRIBUTE -->
-
-	<!-- FOOTER -->
-	<footer class="container-fluid">
-		<div class="container mt-3">
-			<div class="mx-0">
-				<div class="d-flex flex-column flex-md-row align-items-center">
-				<div class="mb-3 pr-md-3">
-					<a class="pr-3" href="https://twitter.com/fairlearn" target="_blank">
-						<img class="img-fluid" src="./images/Twitter_Color.svg" alt=""></a>
-					<a class="pr-3" href="https://stackoverflow.com/questions/tagged/Fairlearn" target="_blank">
-						<img class="img-fluid" src="./images/Stack_Overflow_Color.svg" alt=""></a>
-					<a class="pr-3" href="https://gitter.im/Fairlearn/community?source=orgpage" target="_blank">
-						<img class="img-fluid" src="./images/Gitter_Color.svg" alt=""></a>
-					<a class="pr-3" href="https://github.com/fairlearn/fairlearn" target="_blank">
-						<img class="img-fluid" src="./images/Github_Color.svg" alt=""></a>
-					#fairlearn
-				</div>
-				<div class="flex-grow-1 text-md-right text-nowrap mb-3">
-					<p class="my-0">Copyright © 2021 Fairlearn contributors.</p>
-				</div>
-			    </div>
-			</div>
-		</div>
-	</footer>
-	<!-- FOOTER -->
-
-    <!-- Optional JavaScript -->
-    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
-    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
-    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
-    <script src="./js/landing_page.js"></script>
-  </body>
-</html>
+
+<head>
+    <meta charset="utf-8">
+    <meta content="width=device-width, initial-scale=1.0" name="viewport">
+
+    <title>Web-WIP</title>
+    <meta content="" name="description">
+    <meta content="" name="keywords">
+
+    <!-- Favicons -->
+    <!-- <link href="assets/img/favicon.png" rel="icon">
+    <link href="/assets/img/apple-touch-icon.png" rel="apple-touch-icon"> -->
```

Comment:
Is this needed?

Reply:
Can you provide me your own favicon? thx

Reply:
I mean this is commented out. Is it needed?